### ISSUE TITLE
Focus ring doesn't show up on highlighted items

### ISSUE THREAD

**Issue by talimarcus** [vaadin/px-data-grid#469]
### Steps to reproduce
1. Use the keyboard to navigate to a highlighted item in the data-grid

**Expected behavior:** The item should have a focus ring around it.
**Actual behavior:** The item has focus, but the focus ring is not visible.

### Discussion
When an item is highlighted, the focus ring doesn’t show up when tabbing to that item. This causes the user to get lost if there are adjacent highlighted items. Note that when striping is on, you can see the focus ring on both the white and the gray rows.

### Version
px-data-grid: v1.1.4
Browsers: Chrome 64, Firefox 57, Safari 11

### Screenshot
**"Jeffrey" item has focus**
![screen shot 2018-01-23 at 4 53 37 pm](https://user-images.githubusercontent.com/9092541/35314848-b43550ae-007d-11e8-8f33-cb696b17d8d6.png)
**"Alma" item has focus**
![screen shot 2018-01-23 at 4 53 31 pm](https://user-images.githubusercontent.com/9092541/35314850-b458f77a-007d-11e8-8cb0-a029cebaf4e6.png)
**"Hamilton" item has focus but no focus ring**
![screen shot 2018-01-23 at 4 53 44 pm](https://user-images.githubusercontent.com/9092541/35314847-b41aee58-007d-11e8-952d-26fa3429d113.png)
